metricsMiddleware.js

```diff
@@ -0,0 +1,33 @@
+const createLogger = require('./logger');
+const onFinished = require('on-finished');
+const { removeTokensFromUrl } = require('./lib/utils');
+
+/**
+ * metricsMiddleware
+ * Logging middleware which logs to stdout for metrics tracking
+ *
+ * @param {Object.<String>}
+ * @param {Object.<String>}
+ * @return {Function}
+ */
+module.exports = function metricsMiddleware(options) {
+  const logger = createLogger(options);
+
+  return function logMetrics(req, res, next) {
+    const url = removeTokensFromUrl(req.url);
+
+    onFinished(res, () => {
+      const timestamp = new Date().toJSON();
+      const info = {
+        buffermetrics: true,
+        timestamp,
+        url,
```

Comment:
Should we log the route in addition to the full url so we can remove any query string params?